devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -173,11 +173,22 @@ private Map getBatteryResult(rawValue) {
 		} else {
 			def minVolts = 2.4
 			def maxVolts = 2.7
-			def pct = (volts - minVolts) / (maxVolts - minVolts)
-			def roundedPct = Math.round(pct * 100)
-			if (roundedPct <= 0)
-				roundedPct = 1
-			result.value = Math.min(100, roundedPct)
+			def curValVolts = Math.round(10 * (((Integer.parseInt(device.currentState("battery")?.value) ?: 100) / 100.0)  * (maxVolts - minVolts) + (minVolts))) / 10.0
+			// Only update the battery reading if we don't have a last reading,
+			// OR we have received the same reading twice in a row
+			// OR we don't currently have a battery reading
+			// OR the value we just received is at least 2 steps off from the last reported value
+			if(state?.lastVolts == null || state?.lastVolts == volts || device.currentState("battery")?.value == null || Math.abs(curValVolts - volts) > 0.1) {
+				def pct = (volts - minVolts) / (maxVolts - minVolts)
+				def roundedPct = Math.round(pct * 100)
+				if (roundedPct <= 0)
+					roundedPct = 1
+				result.value = Math.min(100, roundedPct)
+			} else {
+				// Don't update as we want to smooth the battery values
+				result = null
+			}
+			setState(["lastVolts": volts])
```

Comment:
Oh I can make that change.  It's not custom, it's in the base class, but I don't need to use it here, sort of a remnant of an earlier version.